README.md

```diff
@@ -235,6 +230,4 @@ env WINIT_UNIX_BACKEND=x11 alacritty
 Alacritty is released under the [Apache License, Version 2.0].
 
 [Apache License, Version 2.0]: https://github.com/jwilm/alacritty/blob/master/LICENSE-APACHE
-[faq]: https://github.com/jwilm/alacritty#faq
 [tmux]: https://github.com/tmux/tmux
-[Wayland meta issue]: https://github.com/tomaka/winit/issues/306
```

Comment:
Do any of these links actually do anything? I don't think it works this way in markdown?

Reply:
They do work.

Reply:
Oh yeah, I got it. I thought it would try to link to one of the `tmux` mentions, but the `tmux` link is actually for the `terminal multiplexer` string.